src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -128,6 +129,9 @@ public ClientDevicesAuthService(Topics topics, GroupManager groupManager,
         SessionCreator.registerSessionFactory("mqtt", mqttSessionFactory);
         certificateManager.updateCertificatesConfiguration(new CertificatesConfig(getConfig()));
         sessionManager.setSessionConfig(new SessionConfig(getConfig()));
+
+        // Start use cases, so they could be invoked anywhere on the app.
+        UseCases.init(getContext());
```

Comment:
This just starts the useCases with the context so we can invoke them anywhere in the app